src/test/lib/logical_query_plan/lqp_utils_test.cpp

```diff
@@ -298,38 +301,62 @@ TEST_F(LQPUtilsTest, LQPInsertAboveNode) {
 }
 
 TEST_F(LQPUtilsTest, CollectSubqueryExpressionsByLQPNestedSubqueries) {
-  // Prepare an LQP with multiple subqueries in a nested manner
+  // Prepare an LQP with multiple subqueries in a nested manner.
 
-  // clang-format off
-  const auto nested_subquery_lqp =
-  AggregateNode::make(expression_vector(), expression_vector(max_(a_a)),
-    node_a);
-  const auto max_a_subquery = lqp_subquery_(nested_subquery_lqp);
+  for (const auto only_correlated_subqueries : {true, false}) {
+    auto message = std::stringstream{};
+    message << "Consider only uncorrelated subqueries: " << std::boolalpha << only_correlated_subqueries;
```

Comment:
```
SCOPED_TRACE(std::string{"Consider only uncorrelated subqueries: "} + (only... ? "true" : "false"));
```
?

Reply:
Not that it would matter ...

Reply:
actually a debug artifact. Keep it as `SCOPED_TRACE` or remove?